### ISSUE TITLE
/bin/sh: /Applications/System: No such file or directory

### ISSUE THREAD

**Issue by daitarn** [felixhageloh/weather-widget#14]
Hi,
as stated in the title I'm getting this error:
/bin/sh: /Applications/System: No such file or directory
I have Mavericks installed (10.9.5), and I have on my root directory the hidden folder "bin" with the Unix executable named "sh" inside!?
Thanks in advance

**Comment by felixhageloh** [comment]
The `/bin/sh` is just telling that the error uncured when running a shell command. The problem is probably the second part: `/Applications/System/`. It almost looks like it is a cut off path that had a space in it. Does it look like the starting path to your widgets folder maybe?

**Comment by daitarn** [comment]
Thanks for your replay,
no, the widgets folder is in the default location, user Library/App support/Übersicht/widgets. But there is no Applications folder with System in it on my HD, I've searched with System files+Invisible files enabled, strange.. I have 3 other widgets and they are working fine.

**Comment by felixhageloh** [comment]
Are you using the weather widget as is, or did you add some modifications?

**Comment by daitarn** [comment]
As is, I always just copy them first to the widgets folder to see if they work. Maybe some other process on my computer is interfering with your widget or the Übersicht app itself?

**Comment by felixhageloh** [comment]
Yeah it might be something specific to your machine. I don't think it is another process though, but more likely some different configuration. I will see if I can find what it could be!
